src/sign-in-xplat/MainPage.xaml.cs

```diff
@@ -48,36 +52,36 @@ private async void OnSignInClicked(object sender, EventArgs e)
+
+                    try
                     {
-                        IEnumerable<IAccount> accounts = await s_publicClientApp.GetAccountsAsync().ConfigureAwait(false);
-                        IAccount firstAccount = accounts.FirstOrDefault();
-
-                        try
-                        {
-                            // Signs in the user and obtains an Access token for MS Graph
-                            s_authResult = await s_publicClientApp.AcquireTokenSilent(s_scopes, firstAccount)
-                                                              .ExecuteAsync();
-                        }
-                        catch (MsalUiRequiredException ex)
-                        {
-                            // A MsalUiRequiredException happened on AcquireTokenSilentAsync. This indicates you need to call AcquireTokenAsync to acquire a token
-                            Debug.WriteLine($"MsalUiRequiredException: {ex.Message}");
-
-                            s_authResult = await s_publicClientApp.AcquireTokenInteractive(s_scopes)
-#if ANDROID
-                                                              .WithParentActivityOrWindow(Microsoft.Maui.Essentials.Platform.CurrentActivity)
-#endif
-                                                              .ExecuteAsync()
-                                                              .ConfigureAwait(false);
-                        }
+                        // Signs in the user and obtains an Access token for MS Graph
+                        s_authResult = await s_publicClientApp.AcquireTokenSilent(s_scopes, firstAccount)
+                                                            .ExecuteAsync();
+                    }
+                    catch (MsalUiRequiredException ex)
+                    {
+                        // A MsalUiRequiredException happened on AcquireTokenSilentAsync. This indicates you need to call AcquireTokenAsync to acquire a token
+                        Debug.WriteLine($"MsalUiRequiredException: {ex.Message}");
 
-                        requestMessage.Headers.Authorization = new AuthenticationHeaderValue("bearer", s_authResult.AccessToken);
-                    }));
+                        s_authResult = await s_publicClientApp.AcquireTokenInteractive(s_scopes)
+                                                            .ExecuteAsync()
+                                                            .ConfigureAwait(false);
+                    }
+                    
+                    return new AuthenticationHeaderValue("bearer", s_authResult.AccessToken);
+                }
 
                 // Call the /me endpoint of Graph
-                User graphUser = await graphClient.Me.Request().GetAsync();
+                s_httpClient.DefaultRequestHeaders.Authorization = await SignInUserAndGetTokenUsingMSAL();
+                User graphUser = await s_httpClient.GetFromJsonAsync<User>($"{s_graphURL}/me");

```

Comment:
Can we move the url to just be inline here and remove `s_graphURL`.

```suggestion
                User graphUser = await s_httpClient.GetFromJsonAsync<User>("https://graph.microsoft.com/v1.0/me");

```

Reply:
good thing to do @ckittel 

done | addressed from [b593f](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/17/commits/b593f5bba7ce57518e2581911984e0e926347ea4)